front/app/containers/Admin/users/UserTableRow.tsx

```diff
@@ -42,6 +48,55 @@ interface Props {
   authUser: GetAuthUserChildProps;
 }
 
+const getStatusMessage = (user: IUserData): MessageDescriptor => {
+  const highestRole = user.attributes.highest_role;
+  let statusMessage = messages.registeredUser;
+  if (['admin', 'super_admin'].includes(highestRole)) {
+    statusMessage = messages.platformAdmin;
+  } else if (highestRole === 'project_folder_moderator') {
+    statusMessage = messages.folderAdmin;
+  } else if (highestRole === 'project_moderator') {
+    statusMessage = messages.projectManager;
```

Comment:
Shouldn't we stick to old names in the code to be consistent? So, it'll be `messages.folderModerator` and `messages.projectModerator`. Or to rename old occurrences to new names.

Reply:
In this case I was going for the keys being similar to the messages used there.

```
  projectManager: {
    id: 'app.containers.Admin.Users.projectManager',
    defaultMessage: 'Project manager',
  },
```

Not sure what is worse. It would have been nice if we could align the text and the naming but In this case I think it isn't too bad. It should be easy to understand when reading the code below. The person can pick out the role and the message attached to it

```
const getStatusMessage = (user: IUserData): MessageDescriptor => {
  const highestRole = user.attributes.highest_role;
  const roleMessage = { 
    admin: messages.platformAdmin,
    super_admin: messages.platformAdmin,
    project_folder_moderator: messages.folderAdmin,
    project_moderator: messages.projectManager,
    user: messages.registeredUser,
  }; 
  
  return roleMessage[highestRole];
};
```

Reply:
Yes, makes sense.